.gitignore

```diff
@@ -38,6 +38,12 @@ obj/
 *.log
 *.idb
 *.opendb
+
 WSATestCertificate.pfx
 WSATestCertificate.pfx.meta
+
+# ============================= #
+# Custom Project Specific Paths #
+# ============================= #
 /Assets/HoloToolkit-Examples/FastConfigurableShader/Scenes/FastConfigurableTest
```

Comment:
This didn't really address my previous comment. What's being ignored with these two folders that can't just be removed from the repo (like you did with the SharingWithUNET lighting data)?

Reply:
The light data.

Reply:
I don't seem to have that file, and it doesn't seem to be generated at any point? Why the need to both delete and ignore the folder?

/Assets/HoloToolkit-Examples/FastConfigurableShader/Scenes/FastConfigurableTest doesn't exist.

Reply:
yeah it doesn't exist because it was probably already ignored before you ever had a copy of it.

Reply:
>I don't seem to have that file, and it doesn't seem to be generated at any point?

User generates it when they open the scene and build the lighting.

> Why the need to both delete and ignore the folder?

I didn't delete it.  Just marked it as untracked.

Reply:
How do I build the lighting? I've built both scenes a couple times, and I don't see the lighting assets.

Reply:
https://docs.unity3d.com/Manual/GlobalIllumination.html

Reply:
So building lighting is a manual step? Why do we need to ignore these folders then, instead of just making sure the people that build lighting don't check in the asset? This seems like a problem that could expand to needing to add every scene's folder to the .gitignore.

Reply:
It _could_ be set to automatic.

Reply:
I'm all fine with straight up removing it.